src/main/java/net/minestom/server/potion/TimedPotion.java

```diff
@@ -2,22 +2,9 @@
 
 import org.jetbrains.annotations.NotNull;
 
-public class TimedPotion {
-
-    private final Potion potion;
-    private final long startingTime;
-
-    public TimedPotion(@NotNull Potion potion, long startingTime) {
-        this.potion = potion;
-        this.startingTime = startingTime;
-    }
-
-    @NotNull
-    public Potion getPotion() {
+public record TimedPotion(Potion potion, long startingTicks) {
+    @Override
+    public @NotNull Potion potion() {
```

Comment:
seem unnecessary to explicitly declare

Reply:
Absolutely, completely overlooked that from IntelliJ auto converter